tests/php/Forms/SearchableDropdownTraitTest.php

```diff
@@ -261,4 +262,55 @@ public function testLazyLoadedDoesntCallGetSource(string $fieldClass, string $me
         $mockField->setForm(new Form());
         $mockField->$methodToCall();
     }
+
+    public static function provideSearchDataObjectWithMethodForLabelField(): array
+    {
+        return [
+            'single' => [
+                'term' => 'alex',
+                'expected' => '[{"value":0,"label":"Aziel, Alex"}]',
+            ],
+            'multi' => [
+                'term' => 'z',
+                'expected' => '[{"value":0,"label":"Aziel, Alex"},{"value":0,"label":"Cazton, Cindy"}]',
+            ],
+            'email' => [
+                'term' => '@example.com',
+                'expected' => implode('', [
+                    '[{"value":0,"label":"Aziel, Alex"},{"value":0,"label":"Brown, Bob"}',
+                    ',{"value":0,"label":"Cazton, Cindy"}]',
+                ]),
+            ],
+        ];
+    }
+
+    /**
+     * Member has a method getTitle() that returns "Surname, FirstName"
+     * The default label field on the field is "Title", which doesn't exist on the Member database table
+     * It should fall back to using search context in this scenario
+     *
+     * @dataProvider provideSearchDataObjectWithMethodForLabelField
+     */
+    public function testSearchDataObjectWithMethodForLabelField(string $term, string $expected): void
+    {
+        $data = [
+            ['FirstName' => 'Alex', 'Surname' => 'Aziel', 'Email' => 'aaron.aziel@example.com'],
+            ['FirstName' => 'Bob', 'Surname' => 'Brown', 'Email' => 'bob.brown@example.com'],
+            ['FirstName' => 'Cindy', 'Surname' => 'Cazton', 'Email' => 'cindy.cazton@example.com'],
+        ];
+        foreach ($data as $row) {
+            $member = new Member($row);
+            $member->write();
+        }
+        // 4 members because of the 3 members above and the default admin member
```

Comment:
We could just use a getter from `Team` as the label instead of adding `Member` to the mix but I won't delay merging if this is left as-is

Reply:
Leave as is